specs/eventing/motivation.md

```diff
@@ -1,49 +1,33 @@
 # Motivation
 
-The goal of Knative Eventing is to define common, composable primitives to
-enable late-binding event sources and event consumers.
+The goal of the Knative Eventing project is to define common primitives to
+enable composing event-processing applications through configuration, rather
+than application code.
 
-<!-- TODO(n3wscott): [Why late-binding] -->
+Building by combining independent components provides a number of benefits for
+application designers:
 
-Knative Eventing has following principles:
-
-1. Services are loosely coupled during development and deployed independently on
-   a variety of platforms (Kubernetes, VMs, SaaS or FaaS).
+1. Services are loosely coupled during development and may be deployed
+   independently on a variety of platforms (Kubernetes, VMs, SaaS or FaaS). This
+   composability allows re-use of common patterns and building blocks, even
+   across programming language and tooling boundaries.
 
 1. A producer can generate events before a consumer is listening, and a consumer
    can express an interest in an event or class of events that is not yet being
-   produced.
+   produced. This allows event-driven applications to be evolve over time
+   without needing to closely coordinate changes.
 
-1. Services can be connected to create new applications
-   - without modifying producer or consumer.
+1. Services can be connected to create new applications:
```

Comment:
not sure what this one is trying to say. Do you mean we can add new sinks into the eventing infra/mesh w/o impacting existing producers and consumers?